x-pack/plugins/maps/public/connected_components/mb_map/draw_control/draw_control.tsx

```diff
@@ -114,6 +114,7 @@ export class DrawControl extends Component<Props> {
 
   _updateDrawControl() {
     if (!this.props.drawShape) {
+      this._removeDrawControl();
```

Comment:
Can you explain why this change was added? I am not following how this is related to the issue.

Reply:
This fixes the issue that Tomas mentioned above - drawing is disabled, but it is still showing the crosshair

Reply:
I don't think we want to call _removeDrawControl whenever props.drawShape is null. _removeDrawControl has a lot of other side effects like unregistering callbacks and removing the draw control itself. 

Instead, just remove the if statement at line 116. _updateDrawControl properly handles props.drawShape as null and will property set curser and draw mode to SIMPLE_SELECT.